CHANGE_LOG.md

```diff
@@ -2,9 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
-# Chat Widget
+# Chat-Widget
```

Comment:
Is this change a must?

Reply:
yes. it is, in order to automatically update the change_log, I need an trustable anchor point, its easier to find "chat-widget" than "chat widget" as a token word